### DISCUSSION TITLE
How to get the output from 'println!()' in setup or main

### DISCUSSION THREAD

**Discussion by zhibisora** [tauri-apps/tauri#5094] [Q&A]
![image](https://user-images.githubusercontent.com/73344387/187330485-3e75de3f-8e4b-43ad-9fce-3db6631c3e8a.png)

**Comment by abisar** [comment]
https://www.google.com/search?q=pathbuf+to+string+rust

**Reply by zhibisora** [reply]
I got it, thanks!
